docs/user/AnsibleCloudAuthentication.md

```diff
@@ -0,0 +1,47 @@
+# Cloud Authentication
```

Comment:
Cloud Provider Authentication

** Note to developers. Our customer facing Viya Deployment Guide us the term 'cloud provider' vs simply cloud. To be consistent, we should as well.

Reply:
updated